source/Publish Profiles/Pre-production/azure-load-test-reporting.publish.xml

```diff
@@ -102,5 +102,8 @@
     <SqlCmdVariable Include="ReportServer">
       <Value>ReportServer</Value>
     </SqlCmdVariable>
+    <SqlCmdVariable Include="NTBS_URL">
+      <Value>https://ntbs-test.e32846b1ddf0432eb63f.northeurope.aksapp.io/</Value>
```

Comment:
This ought to be pointing at load test (not that it really matters, since load test doesn't have a reporting environment)